### DISCUSSION TITLE
重装后无法登陆专业版

### DISCUSSION THREAD

**Discussion by xiaolin6456** [getlantern/lantern-forum-cn#750] [General]
![微信图片_20240727111127](https://github.com/user-attachments/assets/85f03bea-c441-4b30-936a-9bd5b2c4394e)
您好这是我的附件凭证和邮箱xiaolin6456@sina.com
如何登陆专业版

**Comment by MaxMKMax** [comment]
@xiaolin6456 您好. 请问您有收到激活码吗? 
您可以参考这里怎么登陆:
https://lantern.io/zh-Hans/faq#use

**Reply by xiaolin6456** [reply]
您好，我没有收到激活码

**Reply by MaxMKMax** [reply]
您不需要激活码了. 已经是专业版了.
Device linking code/ 设备链接码:
613799
